.github/workflows/main.yml

```diff
@@ -0,0 +1,56 @@
+name: CI/CD
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ["**"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      # Setup Go
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
```

Comment:
go.mod uses: "go 1.23.0" and "toolchain go1.24.2", should this variable reflect that?